test/shopify_api/rest/product_test.exs

```diff
@@ -19,9 +19,11 @@ defmodule ShopifyAPI.REST.ProductTest do
   end
 
   test "", %{bypass: bypass, shop: _shop, auth_token: token} do
+    product = %{"product_id" => "_", "title" => "Testing Create Product"}
+
     products = %{
       "products" => [
```

Comment:
🛠 Would recommend swapping this around a bit - we only need to consider the map wrapper in our `Bypass` callback:

```elixir
product = %{"id" => "1", "title" => "Test Product"}

Bypass.expect_once(bypass, "GET", "...", fn conn ->
  {:ok, body} = JSONSerializer.encode(%{products: [product]})
  Conn.resp(conn, 200, body)
end)

assert {:ok, [^product]} = Product.all(token)
```